### PR TITLE
fix(core): get radio default value like checkboxes

### PR DESCRIPTION
**Pull request by ianwremmel**

### CODE REVIEW

File: src/lib/vueify.js [hunk 1]
```diff
@@ -91,7 +91,10 @@ function generateModel(el, data) {
  * @returns {boolean|string|null}
  */
 function generateModelForInput(el) {
-  if (el.getAttribute('type') === 'checkbox') {
+  if (
+    el.getAttribute('type') === 'checkbox' ||
+    el.getAttribute('type') === 'radio'
```
Comment (bzang) [inline src/lib/vueify.js]: Is this a better way to do it?  unless `.includes()` is available.
```suggestion
    ['checkbox', 'radio'].indexOf(el.getAttribute('type')) >= 0
```

File: features/server/views/pages/form.ejs [hunk 2]
```diff
@@ -21,12 +21,12 @@
...
         <div>
-          <input selected="selected" readonly="readonly" type="radio" value="false" checked="checked" name="options" id="option_false" />
-          <label for="option_false">Option False</label>
+          <input selected="selected" checked="checked" type="radio" value="true" name="options" id="option_true" />
+          <label for="option_true">Option True</label>
         </div>
         <div>
-          <input type="radio" value="true" name="options" id="option_true" />
-          <label for="option_true">Option True</label>
+          <input selected="selected" readonly="readonly" type="radio" value="false" checked="checked" name="options" id="option_false" />
```
Comment (bzang) [inline features/server/views/pages/form.ejs]: isn't `selected` not a valid attribute for radio buttons?

Comment (ianwremmel) [reply features/server/views/pages/form.ejs]: simple form adds both, so i wanted the test to include it. The only change here is to put the default second so the test is actually valid.

File: src/lib/vueify.js (hunk 1 above)
Comment (ianwremmel) [reply src/lib/vueify.js]: good call.

**Pull request merged**